site/cds_rdm/components.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2025 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS RDM service components."""
+
+from celery import shared_task
+from flask import current_app
+from invenio_access.permissions import system_identity
+from invenio_communities.proxies import current_communities
+from invenio_drafts_resources.services.records.components import ServiceComponent
+from invenio_i18n import lazy_gettext as _
+
+from invenio_rdm_records.services.errors import ValidationErrorWithMessageAsList
+from invenio_rdm_records.proxies import current_rdm_records
+from invenio_records_resources.services.uow import TaskOp
+
+
+@shared_task()
+def create_community_inclusion_request(record_id):
+    """Create a community inclusion request for ."""
+    # Create a community-inclusion request
+    sis_community_id = current_app.config.get("CDS_CERN_SCIENTIFIC_COMMUNITY")
```

Comment:
I would avoid `sis` anywhere, in case the group name changes in the future.
Acronym, or `library_managed_comm_id`?
```suggestion
    csc_community_id = current_app.config.get("CDS_CERN_SCIENTIFIC_COMMUNITY")
```